autopilot/contractor.go

```diff
@@ -1154,16 +1154,16 @@ func (c *contractor) renewContract(ctx context.Context, w Worker, ci contractInf
 		return api.ContractMetadata{}, false, errors.New("insufficient budget")
 	}
 
-	// calculate the host collateral
-	endHeight := endHeight(cfg, state.period)
-	expectedStorage := renterFundsToExpectedStorage(renterFunds, endHeight-cs.BlockHeight, settings)
-	newCollateral := rhpv2.ContractRenewalCollateral(rev.FileContract, expectedStorage, settings, cs.BlockHeight, endHeight)
-
 	// sanity check the endheight is not the same on renewals
+	endHeight := endHeight(cfg, state.period)
 	if endHeight == rev.EndHeight() {
```

Comment:
Can we make this `<=`? Do be extra safe?

Reply:
sure thing!